data_dir/cs_multidc_si_lcs.yaml

```diff
@@ -56,8 +56,8 @@ queries:
     cql: select * from keyspace1.standard2 where key=?
     fields: samerow
   si_read1:
-    cql: select * from keyspace1.sec_ind_c2 where c2 = ? LIMIT 1
+    cql: select * from keyspace1.standard2 where c2 = ? LIMIT 1
     fields: samerow
   si_read2:
-    cql: select * from keyspace1.sec_ind_c3 where c3 = ? LIMIT 1
+    cql: select * from keyspace1.standard2 where c3 = ? LIMIT 1
```

Comment:
I assume, originally intention is select from index view. Am I correct @aleksbykov ?

It is not same, as filter by different columns. But it should fail because of "c2" and "c3" are not part of PK so we can filter with ALLOW FILTERING only.

But if you want to select from index view, I suppose their names are "sec_ind_c2_index" and "sec_ind_c3_index"

Reply:
I fixed my previous comment

Reply:
yes the was to read from index, but when it was sec_ind_c2_index, the c-s always failed that such table is missing. I checked manually and verify in scylla doc, to read by new index column this syntax should be used. base table with column as new index

Reply:
If I understand correctly your question, filter by "c2" or "c3" columns is reading from index view actually

Reply:
yes, but in latest change (that was just merged), we start now to read from `keyspace1.standard2` instead of `keyspace1.sec_ind_c[2|3]`
so we stopped reading from index, right? if we did stop, then it is not good, as the file is supposed to use SI (by what the filename suggests)

Reply:
No, filter by "c2" or "c3" columns is allowed if those columns are part of the index. To filter by those columns means to use the view. And this is correct way to use SI. Index is created to allow filtering by non-PK columns. It is realised via view mechanism. But it does not mean that user should select from view. View does not keep all data that kept in the base table. User still should select from base table, exactly on the way how Alex fixed. 
In previous version Alex tried to select from index view and it failed because of `name of view` != `SI name`.